lib/licensee/project_file.rb

```diff
@@ -1,18 +1,18 @@
+require 'charlock_holmes'
+
 module Licensee
   class Project
     class File
       attr_reader :content, :filename
 
-      ENCODING = Encoding::UTF_8
-      ENCODING_OPTIONS = {
-        invalid: :replace,
-        undef:   :replace,
-        replace: ''
-      }.freeze
-
       def initialize(content, filename = nil)
         @content = content
-        @content.encode!(ENCODING, ENCODING_OPTIONS)
+        unless @content.empty?
```

Comment:
I think we need to do better here. Charlock is not free (particularly not the transcoding step). The logic needs to be more like:

```ruby
@content.force_encoding(Encoding::UTF_8)
if !@content.valid_encoding?
  # do the transcoding
end
```